src/nplinker/metabolomics/__init__.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+
+logging.getLogger(__name__).addHandler(logging.NullHandler())
+
+__author__ = "Helge Hecht"
+__email__ = "h.hecht@esciencecenter.nl"
+__version__ = "1.3.2"
```

Comment:
Please remove this part. Let's keep these info only in the top __init__.py file.

Reply:
Removed in https://github.com/NPLinker/nplinker/pull/78/commits/c2a10d7fc5231eae0bed5c2c97d5b5b057644243